app/Models/Game.php

```diff
@@ -99,12 +104,21 @@ protected static function newFactory(): GameFactory
 
     // == logging
 
-    // protected static $recordEvents = ['created'];
+    // TODO log game creation from the connect api
 
     public function getActivitylogOptions(): LogOptions
     {
         return LogOptions::defaults()
-            ->logOnlyDirty();
+            ->logOnly([
+                'Title',
+                'ForumTopicID',
+                'GuideURL',
+                'Publisher',
+                'Developer',
+                'Genre',
+            ])
```

Comment:
In a follow-up PR, I'll need to update the legacy .php endpoint functions to correctly record the causer for these.